tests/test_api_tcp_client.py

```diff
@@ -328,6 +308,13 @@ async def test_breeze_build_command_function_invalid_mode(resource_path_root):
         remote.build_command(DeviceState.ON, ThermostatMode.COOL, 20, ThermostatFanLevel.AUTO, ThermostatSwing.ON, DeviceState.OFF)
 
 
+async def test_breeze_build_command_function_specific_case():
+    remote_manager = SwitcherBreezeRemoteManager()
+    remote_7001 = remote_manager.get_remote("ELEC7001")
+    command = remote_7001.build_command(DeviceState.OFF, ThermostatMode.COOL, 20, ThermostatFanLevel.HIGH, ThermostatSwing.ON, DeviceState.ON)
+    assert_that(command.command).is_equal_to("00000000524337327c32317c33327c32367c34437c39387c537c32327c30337c373237325b32325d7c39383841303030303830")
```

Comment:
It's been a long time since I worked on this project, I don't remember what goes into these packets. Are you there's isn't anything important here?

Reply:
Yes.
As part of this coverage improvement, I just saw that part of a code didn't reach by any test so I added a test to support that.

In this file, There is also part of the code which is not used anywhere..

Reply:
Yes, I noticed that - thanks.
I was only talking about the packet itself, the one you used in the `assert_that` statement, just making sure it doesn't contain any personal/sensitive data before merging.

Reply:
No its not. its the same packet as used before in the code so it should be fine :)